README.md

```diff
@@ -1,24 +1,110 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
-Things you may want to cover:
+> A mobile web application where you can manage your budget
+![image](https://user-images.githubusercontent.com/57562869/143658044-3f0d738f-04f9-4320-b2ba-5d22129b9456.png)
 
-* Ruby version
 
-* System dependencies
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Configuration
+## Built With
 
-* Database creation
+- Ruby on Rails
+- PostgreSQL
 
-* Database initialization
+## Live Link
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+## Getting Started
 
-* Deployment instructions
+To get a local copy up and running follow these simple example steps.
 
-* ...
+### Prerequisites
+
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Rails](https://gorails.com/)
+
+### Setup
+
+- Make sure you have Ruby on Rails set up properly on your computer
+- Clone or download this repo on your machine
+- Enter project directory
+
+### Install
+
+```sh
+bundle install
+```
+
+### Database
+
+```sh
+# Create user
+sudo -u postgres createuser recipe_app -s
+
+# Create the database
+rails db:create
+
+## Apply migration
+rails db:migrate
+
+# Load the schema
+rails db:schema:load
+```
+
+### Run
+
+```sh
+rails s
+```
+
+### Test
+
+```sh
+rspec
+```
+
+### Troubleshoot
+
+```sh
+### Rspec failing
+RAILS_ENV=test rake db:reset
+```
+
+```sh
+### New Scaffold
+rails g scaffold_controller model
+```
+
+```sh
+### Undo migration
+rake db:migrate VERSION=0
+```
+
+## Authors
```

Comment:
- [ ] Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.